src/entities/mongodb.ts

```diff
@@ -57,6 +60,14 @@ const create = <T, Tid extends keyof T>(
       const collection = await db.collection<T, Tid>(collectionName);
       return collection.findOne(criteria);
     },
+    find: async (criteria) => {
+      if (!criteria) {
+        throw new Error('MongoEntity::findBy: criteria must be an object');
+      }
```

Comment:
Consider making the condition more specific, with a error message that matches closely:

```suggestion
      if (typeof criteria !== 'object' || criteria === null)) {
        throw new Error('MongoEntity::findBy: criteria must be a non-null object');
      }
```

Reply:
done!